providers/apis/dydx/parse.go

```diff
@@ -49,6 +50,7 @@ var ProviderMapping = map[string]string{
 	"Raydium":              raydium.Name,
 	"UniswapV3-Ethereum":   uniswapv3.ProviderNames[constants.ETHEREUM],
 	"UniswapV3-Base":       uniswapv3.ProviderNames[constants.BASE],
+	coinmarketcap.Name:     coinmarketcap.Name,
```

Comment:
Needs to be added here since the existing logic inherits this class. Once we migrate to v6 we should move this logic directly into the research provider.